### PR TITLE
lasso: add 2.6.0

### PR DESCRIPTION
**Pull request by grueni**
dependency of mod_auth_mellon

### CODE REVIEW

File: components/library/lasso/Makefile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+#
+
+
+include ../../../make-rules/shared-macros.mk
+
+BITS=64
+PATH=$(PATH.gnu)
```
Comment (Mno-hime) [inline components/library/lasso/Makefile]: Same as with the `xmlsec` PR.

File: components/library/lasso/Makefile [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+BASEURL=	https://dev.entrouvert.org
```
Comment (Mno-hime) [inline components/library/lasso/Makefile]: As it's used only twice, rather avoid it.

File: components/library/lasso/Makefile [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+COMPONENT_NAME=		lasso
+COMPONENT_VERSION=	2.6.0
+COMPONENT_FMRI= library/lasso
+COMPONENT_SUMMARY= Lasso library
```
Comment (Mno-hime) [inline components/library/lasso/Makefile]: Better:
```suggestion
COMPONENT_SUMMARY= Lasso - library implementing the SAML 2.0 and ID-FF 1.2 protocols
```

File: components/library/lasso/Makefile [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+COMPONENT_CLASSIFICATION= System/Libraries
+COMPONENT_SRC=    $(COMPONENT_NAME)-$(COMPONENT_VERSION)
```
Comment (Mno-hime) [inline components/library/lasso/Makefile]: It seems that some space separators in the `COMPONENT_` block are tabs and some spaces, make it consistent, I prefer tab(s).

**Review by Mno-hime — APPROVED** [review]

**Comment by alarcher** [comment]
@grueni Could you try to squash the commits in the future when you are ready with PRs? It is easy to miss that I need to squash on the phone...

**Pull request merged**